mParticle-RevealMobile/MPKitRevealMobile.h

```diff
@@ -17,15 +17,20 @@
 //
 
 #import <Foundation/Foundation.h>
-#import "MPKitExecStatus.h"
-#import "MPKitProtocol.h"
+#import <mParticle_Apple_SDK/MPKitExecStatus.h>
```

Comment:
Can we use `#import "MPKitExecStatus.h"` and `#import "MPKitProtocol.h"` instead?